tests/e2e/normalize.go

```diff
@@ -216,6 +216,20 @@ func normalizeKRMObject(t *testing.T, u *unstructured.Unstructured, project test
 	// Specific to DataPlex
 	visitor.replacePaths[".status.observedState.metastoreStatus.updateTime"] = "2024-04-01T12:34:56.123456Z"
 	visitor.replacePaths[".status.observedState.assetStatus.updateTime"] = "2024-04-01T12:34:56.123456Z"
+	visitor.replacePaths[".status.observedState.executionStatus.updateTime"] = "2024-04-01T12:34:56.123456Z"
```

Comment:
nit: there is an ongoing effort on moving normalization into each individual mock packages. e.g.
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/mockgcp/mocktpu/normalize.go